nat-lab/tests/telio_features.py

```diff
@@ -3,6 +3,10 @@
 from typing import Optional, List
 
 
+def ExcludeIfNone(value) -> bool:
```

Comment:
This does not have a call site, nor can I find it anywhere in the codebase, could you explain it? thanks!